### PR TITLE
No AATF Choose Site View

### PR DESCRIPTION
**Pull request by NateHogan**
Delivers
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/47076

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+@using EA.Weee.Core.Helpers
+@using EA.Weee.Web.RazorHelpers
+
+@model EA.Weee.Web.Areas.Aatf.ViewModels.SelectYourAatfViewModel
+@{
+    var title = "Which site would you like to manage evidence notes for?";
+    ViewBag.Title = title + " - GOV.UK";
+}
+
+<section id="main-content">
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml:11]: use the section full_main_content

**Comment by JohnPadgettCivica** [comment]
@NateHogan  Small comment and as this is a small PR create the unit test to check the correct view.

File: src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+
+            @Html.ActionLink("Back", "ChooseActivity", "Home", new { area = "Scheme", pcsId = Model.OrganisationId }, new { @class = "govuk-back-link weee-back-link" })
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml:15]: can you use that @(this.WeeeGds().BackLink ..

Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml:15]: chooseActivity seems to be in the Aatf area, judging by the src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml path, is that area="Scheme" correct?

File: src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml (hunk 1 above)
Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml]: Done

File: src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml (hunk 2 above)
Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml]: Done

Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml]: I believe scheme is the correct area, yes

Comment (meglenapnovoselska) [reply src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml]: okay

File: src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+@section full_main_content
+{
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+
+            @(this.WeeeGds().BackLink(@Url.UrlFor<HomeController>(a => a.ChooseActivity(Model.OrganisationId))))
+
+            @Html.Gds().ValidationSummary()
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml:20]: Can remove this :)

**Pull request merged**